### PR TITLE
improvised ballistic plates in the style of scrap metal armor

### LINKED ISSUE #49477: Add makeshift/metalplate ESAPI vest
# Is your feature request related to a problem? Please describe.
You cannot easily modify ESAPI vest in a crude manner

# Describe the solution you'd like
Ability to load an empty ESAPI vest with regular metal plates to make kind of simple and light but not so protective scrappy apocalyptic vest. Not good against projectiles with stats simlar to scrap cuirass

# Describe alternatives you've considered
Make empty vest less protective, as description says its only synthetic fabric

# Additional context
Just thought it would fit the game. Mostly just a scrap cuirass alternative if you dont have the skill but have means to shape some large metal plates in to approriate shape of plate holder compartment. Like by sawing off exces material or bending it in shape with pliers, and just put whatever fits in the compartment 
74535280-old-metal-sheet-roof-texture-pattern-of-old-metal-sheet-metal-sheet-texture-rusty-metal-sheet-textur

### PR DESCRIPTION
**Pull request by delarifert**
#### Summary
Content "Adds improvised metal ballistic plates in the style of scrap armor"

#### Purpose of change
This fixes #49477, or at least implements the content it requests.

Besides implementing a good idea, I'm doing this for exposure to the armor system and the community, to hopefully work on other things too. 

#### Describe the solution
Adds two recipes, scrap ESAPI and scrap ESBI plates, to represent taking a hunk of metal and stuffing it in a plate carrier.

#### Describe alternatives you've considered
I briefly considered not doing this.

#### Testing
I've spawned the new scrap plates in game and checked their values, both stats and in combat using a US ballistic vest. I've also tried both the added recipes.

#### Additional context
This was OK'd in Discord, and I'm new. Please let me know if I need to do anything else or messed up, thanks.

### CODE REVIEW

File: data/json/items/armor/ballistic_armor.json [hunk 1]
```diff
@@ -176,7 +176,7 @@
     "//2": "link like: https://www.shindn.com/products/nij-iiia?variant=35614363582627&currency=USD&utm_medium=product_sync&utm_source=google&utm_content=sag_organic&utm_campaign=sag_organic&utm_campaign=gs-2019-08-03&utm_source=google&utm_medium=smart_campaign&gclid=Cj0KCQjw5-WRBhCKARIsAAId9FlGQAManRGZ0qpV9AIkD23FZfpm4aPDI4PHE9qPktZ2bdQY2NmcIVkaAs2pEALw_wcB",
     "description": "A heavy and concave ballistic plate made of tempered steel, designed to stop small-caliber rounds.  \"NIJ III PROTECTION\" is etched into the back of the plate.  This is intended to be inserted into a ballistic vest, and can't be worn on its own.",
     "weight": "2500 g",
-    "//3": "30cm x 25cm x .55cm",
+    "//3": "30cm x 25cm x .5cm",
```
Comment (bombasticSlacks) [inline data/json/items/armor/ballistic_armor.json:179]: I'm not sure why this was changed?

**Review by bombasticSlacks — COMMENTED** [review]
You'll need to lint your files. Take both the files you have changed and copy the **whole files text** into https://dev.narc.ro/cataclysm/format.html. It will make them look pretty (and is required for merging) then you can just copy the result back into the file.

File: data/json/items/armor/ballistic_armor.json (hunk 1 above)
Comment (delarifert) [reply data/json/items/armor/ballistic_armor.json:179]: huh. must have been a mistake getting confused between my copy pastes. need to undo that line

Comment (bombasticSlacks) [reply data/json/items/armor/ballistic_armor.json:179]: then you can merge it here or do it when you are doing your formatting
```suggestion
    "//3": "30cm x 25cm x .55cm",
```

Comment (delarifert) [reply data/json/items/armor/ballistic_armor.json]: okay, i've linted the files and removed that error

**Pull request merged**